tutorial/practicals/nemo/1_nemo_psyir/README.md

```diff
@@ -173,7 +195,7 @@ interesting from a performance point of view.
 To familiarise yourself with PSyIR navigation, you can:
 
 1. Modify the transformation script so that it breaks-out into the Python
```

Comment:
at line 141 (and everywhere else), we should use "tra_adv.F90" (uppercase F), otherwise pasting the command says: "File not found"

Reply:
Good spot.